[Vectorize] Announcement for the anticipated blocking of legacy index creation (#17593)
cloudflare/cloudflare-docs: src/content/partials/vectorize/vectorize-legacy.mdx

Please use the `wrangler vectorize --deprecated-v1` flag to create, get, list, delete and insert vectors into legacy Vectorize V1 indexes.

> Please note that the creation of legacy Vectorize indexes will be blocked by December 2024. Other operations will remain functional.

Reviewer: ```suggestion
Please note that by December 2024, you will not be able to create legacy Vectorize indexes. Other operations will remain functional.
```

> Please note that the creation of legacy Vectorize indexes would be blocked by December 2024. Other operations would remain functional.

Reviewer 2: ```suggestion
Please note that the creation of legacy Vectorize indexes will be blocked by December 2024. Other operations will remain functional.
```
